Replace '<code>nano</code>' with 'nano'

wking opened the pull request:
The project refers to itself as [“GNU nano”](http://www.nano-editor.org/overview.php), and neither it nor
[Wikipedia](http://en.wikipedia.org/wiki/GNU_nano) use any code-like markup around the project name.

This also fixes some Markdown syntax that leaked into index.html.

wking committed:
index.html: Replace '`nano`' with '<code>nano</code>'

The Markdown version came in with d5b65522 (Clarify that the SWC
Windows installer runs once, 2015-04-02, #193), but index.html is
HTML, not Markdown ;).

wking committed:
Replace '<code>nano</code>' with 'nano'

The project refers to itself as "GNU nano" [1], and neither it nor
Wikipedia [2] use any code-like markup around the project name.

[1]: http://www.nano-editor.org/overview.php
[2]: http://en.wikipedia.org/wiki/GNU_nano

rgaiacs commented:
The same should apply to SQLite?

wking commented:
On Mon, Apr 06, 2015 at 09:16:45PM -0700, Raniere Silva wrote:

> The same should apply to SQLite?

Works for me.  Should I tack that on to this PR tonight, or should I
submit a separate PR after this lands?

rgaiacs commented:
> > The same should apply to SQLite?
> 
> Works for me.  Should I tack that on to this PR tonight, or should I
> submit a separate PR after this lands?

I prefer use this PR.

wking committed:
Replace '<code>bash</code>' with 'Bash' when referring to the project

The project refers to itself as "Bash" [1].

[1]: https://www.gnu.org/software/bash/

wking committed:
Replace '<code>sqlite</code>' with 'SQLite'

The project refers to itself as "SQLite" [1], and none of these
references are suggesting command-line invocation.

[1]: http://www.sqlite.org/

wking commented:
On Tue, Apr 07, 2015 at 11:58:05AM -0700, Raniere Silva wrote:

> > > The same should apply to SQLite?
> > 
> > Works for me.  Should I tack that on to this PR tonight, or should
> > I submit a separate PR after this lands?
> 
> I prefer use this PR.

Done, and for Bash as well.

rgaiacs commented on FAQ.md:
```diff
@@ -43,7 +43,7 @@
 
     We have built a small installation helper for Windows
-    that installs `nano` and `sqlite`, adds R to the path, and so on.
+    that installs nano and `sqlite`, adds R to the path, and so on.
```
`sqlite` -> SQLite

wking replied:
On Tue, Apr 14, 2015 at 12:23:04PM -0700, Raniere Silva wrote:

> > -    that installs `nano` and `sqlite`, adds R to the path, and so on.
> > -    that installs nano and `sqlite`, adds R to the path, and so on.
> 
> `sqlite` -> SQLite

Done thanks.  I'd forgotten I was editing the FAQ too.
